packages/office-ui-fabric-react/src/components/Popup/Popup.tsx

```diff
@@ -66,6 +66,9 @@ export class Popup extends React.Component<IPopupProps, IPopupState> {
         this._originalFocusedElement.focus();
       }
     }
+
+    delete this._originalFocusedElement;
```

Comment:
For good measure I wonder if you should also dispose and de-reference `this._async`. Might also be good to add a comment about why this is needed (at least until the pattern is prevalent enough throughout the codebase that people just follow it).

Reply:
Done! However, I opted not to de-reference `this._async` as I don't think its necessary until I see it pop-up in snapshots. The disposables array is the problematic pattern because it allows for one to store closures without an explicit clean-up call on unmount.